assets/models/system/databricks-dolly-v2-12b/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/databricks-dolly-v2-12b/7dde7ac/mlflow_model_folder
+  container_path: huggingface/databricks-dolly-v2-12b/1693844710/mlflow_model_folder
```

Comment:
Space in the path?

Reply:
As I don't see the Space but Raised a PR for all in the new request.
https://github.com/Azure/azureml-assets/pull/1181